components/cards.css

```diff
@@ -0,0 +1,132 @@
+.display-card{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+.card-text-overlay{
+    position: relative;
+    background-color: var(--offwhite);
+    max-width: 350px;
+}
+
+.card-text-shadow{
+    position: relative;
+    background-color: var(--offwhite);
+    max-width: 350px;
+    box-shadow: 0px 8px 8px 6px rgba(0,0,0,.5);
+}
+
+.card-title-over{
+    color: var(--white);
+    position: absolute;
+    bottom: 148px;
+    right: 8px;
+}
+
+.card-img-container{
+    width: 100%;
+}
+
+.card-img{
+    width: 100%;
+}
+
+.card-body{
+    padding: 1rem;
+    line-height: 1.25rem;
+}
+
+.card-button{
+    display: flex;
+    margin-top: 1rem;
+}
+
+.btn-card{
+    font-weight: bolder;
+    padding: 8px;
+    margin-right: 8px;
+    background-color: var(--white);
+    border: 0;
+}
+
+.btn-card:hover{
+    background-color: var(--primary-light);
+    border-radius: 2px;
+}
+
+.card-action-items{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.card-items{
+    display: flex;
+}
+
+.card-actions{
+    width: 50%;
+    align-items: center;
+    margin-top: 1rem;
+    display: flex;
+    justify-content: space-around;
+}
+
+.body-title-over{
+    font-weight: bold;
+    font-size: 1.2rem;
+    color: var(--white);
+}
+
+.body-subtitle-over{
+    margin-top: 4px;
+    color: var(--white);
+}
+
+.card-body-title{
+    font-weight: bold;
+    font-size: 1.2rem;
+}
+
+.card-body-subtitle{
+    margin-top: 4px;
+    font-weight: bold;
+    color: rgb(95, 92, 92);
+    margin-bottom: 1rem;
+}
+
+.card-horizontal{
+    max-width: 21.875rem;
+    box-shadow: 0px 8px 8px 6px rgba(0,0,0,.5);
+}
+
+.card-horizontal-head{
+    display: flex;
+}
+
+.card-horizontal-img{
+    width: 40%;
+}
+
+.card-horizontal-action-items{
+    padding: .75rem;
+}
+
+.card-horizontal-actions{
+    width: 50%;
+    align-items: center;
+    display: flex;
+    justify-content: space-around;
+}
+
+.card-badge{
+    color: var(--white);
+    background-color: var(--red);
+    padding: .5rem;
+    border-radius: 4px;
+    position: absolute;
+    bottom: 268px;
```

Comment:
use rem instead of pixel else everything looks good from my side.

Reply:
Changed successfully!